webview/qt.py

```diff
@@ -24,7 +24,7 @@
     from PyQt5 import QtCore
 
     # Check to see if we're running Qt > 5.5
-    from QtCore import QT_VERSION_STR
+    from PyQt5.QtCore import QT_VERSION_STR
```

Comment:
Is this semantic sugar or is there more to this change?

Reply:
There is. Again, I don't exactly know why this is needed, but sometimes Python 3
doesn't recognise `QtCore` as a module and throws a false `ImportError` when
QT5 is present. This seems to pass both Python 2.x and 3.6.